docs/inventories.rst

```diff
@@ -0,0 +1,773 @@
+
+The ``host_filter`` parameter allows for:
+
+- grouping via ()
+- use of the boolean and operator:
+
+  - ``__`` to reference related fields in relational fields
+  - ``__`` is used on ``ansible_facts`` to separate keys in a JSON key path
+  - ``[]`` is used to denote a json array in the path specification
+  - ``""`` can be used in the value when spaces are wanted in the value
+
+- "classic" Django queries may be embedded in the ``host_filter``
+
+Examples:
+
+::
+
+  /api/v2/hosts/?host_filter=name=localhost
+  /api/v2/hosts/?host_filter=ansible_facts__ansible_date_time__weekday_number="3"
+  /api/v2/hosts/?host_filter=ansible_facts__ansible_processor[]="GenuineIntel"
+  /api/v2/hosts/?host_filter=ansible_facts__ansible_lo__ipv6[]__scope="host"
+  /api/v2/hosts/?host_filter=ansible_facts__ansible_processor_vcpus=8
+  /api/v2/hosts/?host_filter=ansible_facts__ansible_env__PYTHONUNBUFFERED="true"
+  /api/v2/hosts/?host_filter=(name=localhost or name=database) and (groups__name=east or groups__name="west coast") and ansible_facts__an
+
+You can search ``host_filter`` by host name, group name, and Ansible facts.
+
+The format for a group search is:
+
+::
+
+  groups.name:groupA
+
+The format for a fact search is:
+
+::
+
+  ansible_facts.ansible_fips:false
+
+You can also perform Smart Search searches, which consist a host name and host description.
+
+::
+
+  host_filter=name=my_host
+
+
+If a search term in ``host_filter`` is of string type, to make the value a number (e.g. ``2.66``), or a JSON keyword (e.g. ``null``, ``true`` or ``false``) valid, add double quotations around the value to prevent AWX from mistakenly parsing it as a non-string:
+
+::
+
+  host_filter=ansible_facts__packages__dnsmasq[]__version="2.66"
+
+
+.. _ug_host_filter_facts:
+
+Define host filter with ``ansible_facts``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To use ``ansible_facts`` to define the host filter, perform the following steps:
+
+1. Anywhere there is a **Smart host filter** field, click the |search| button next to it to open a pop-up window to filter hosts for this inventory.
+
+2. In the search pop-up window, change the search criteria from **Name** to **Advanced** and select **ansible_facts** from the **Key** field.
+
+
+If you wanted to add an ansible fact of
+
+::
+
+    /api/v2/hosts/?host_filter=ansible_facts__ansible_processor[]="GenuineIntel"
+
+
+In the search field, enter ``ansible_processor[]="GenuineIntel"`` (no extra spaces or ``__`` before the value) and press **[Enter]**.
+
+.. image:: _static/images/inventories-smart-define-host-filter-facts.png
+   :alt: Example of an advanced search for host filter Ansible facts
```

Comment:
I think `.. image::` directives should be replaced with `.. figure::` per https://docutils.sourceforge.io/docs/ref/rst/directives.html#figure. This would allow having image captions, not just alt text.

Reply:
Ok, let's do this one in another PR but can we just leave it since it would affect not just here but affect other docs in the repo that have already been approved and published.

Reply:
Yes, we can treat this as out of the scope of this PR.

Reply:
thank you!

Reply:
File an issue for this, then. And self-assign.